### PR TITLE
removed syntax high light and added code mirror for showing YAML

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
image

### CODE REVIEW

File: src/webview/common/devfileSearch.tsx [hunk 1]
```diff
@@ -74,6 +76,64 @@ function LinkButton(props: { href: string; disabled: boolean; onClick: () => voi
     );
 }
 
+/*const vscodeTheme = createTheme({
```
Comment (datho7561) [inline src/webview/common/devfileSearch.tsx:79]: If you aren't using this code, please remove it

**Review by datho7561 — CHANGES_REQUESTED** [review]
One small change, but otherwise looks good

**Comment by msivasubramaniaan** [comment]
> One small change, but otherwise looks good

Sorry thought of remove it, somehow missed

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks, Muthu!

**Pull request merged**